python/plugins/db_manager/db_plugins/oracle/info_model.py

```diff
@@ -624,7 +624,7 @@ def spatialInfo(self):
         if self.table.extent and len(self.table.extent) == 4:
             extent_str = (u"{:.9f}, {:.9f} - {:.9f}, "
                           u"{:.9f}".format(*self.table.extent))
-        elif self.table.rowCount > 0 or self.table.estimatedRowCount > 0:
+        elif (self.table.rowCount is not None and self.table.rowCount > 0) or (self.table.estimatedRowCount is not None and self.table.estimatedRowCount > 0):
```

Comment:
same comment than above

Reply:
Here, I guess it can be replaced as you said, since you don't have the '>=', only '>'.